tests/FSharp.Compiler.ComponentTests/EmittedIL/Structure/FloatsAndDoubles.fs.il.debug.bsl

```diff
@@ -65,8 +65,9 @@
     .method public hidebysig specialname 
             instance float64  get_F() cil managed
     {
-      .custom instance void [runtime]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = ( 01 00 00 00 )
-      .custom instance void [runtime]System.Diagnostics.DebuggerNonUserCodeAttribute::.ctor() = ( 01 00 00 00 )
+      .custom instance void [mscorlib]System.Runtime.CompilerServices.IsReadOnlyAttribute::.ctor() = ( 01 00 00 00 ) 
```

Comment:
The diff shows that the attribute was added here, was not present in this test before.
At the same time, it is pointing to the attribute from [mscorlib].

Do we know why it was missing before?

Reply:
The missing lines for:
````
      .custom instance void [runtime]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = ( 01 00 00 00 )
      .custom instance void [runtime]System.Diagnostics.DebuggerNonUserCodeAttribute::.ctor() = ( 01 00 00 00 )
````
is a bit iffy though.

Reply:
Okay, I understand the baseline file differences.  We recently checked in some codegen changes to DebuggerNonUserCodeAttribute and only updated the release version of the baselines .

Sigh!!!!!

Reply:
> The diff shows that the attribute was added here, was not present in this test before.
> At the same time, it is pointing to the attribute from [mscorlib].
> 
> Do we know why it was missing before?
> 

This pr reintroduces generating of those attributes, I had to turn it off due to a bug.